Index: src/main/java/it/polimi/ingsw/galaxytruckers/view/enums/States.java
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/galaxytruckers/view/enums/States.java b/src/main/java/it/polimi/ingsw/galaxytruckers/view/enums/States.java
deleted file mode 100644
--- a/src/main/java/it/polimi/ingsw/galaxytruckers/view/enums/States.java	(revision 61c0b0884c4e17776f1d02353a8d83e89fed06d3)
+++ /dev/null	(revision 61c0b0884c4e17776f1d02353a8d83e89fed06d3)
@@ -1,5 +0,0 @@
-package it.polimi.ingsw.galaxytruckers.view.enums;
-
-public enum States {
-    BUILDING, ADVENTURE
-}
Index: src/main/java/it/polimi/ingsw/galaxytruckers/view/enums/RotationDirection.java
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/galaxytruckers/view/enums/RotationDirection.java b/src/main/java/it/polimi/ingsw/galaxytruckers/view/enums/RotationDirection.java
deleted file mode 100644
--- a/src/main/java/it/polimi/ingsw/galaxytruckers/view/enums/RotationDirection.java	(revision 61c0b0884c4e17776f1d02353a8d83e89fed06d3)
+++ /dev/null	(revision 61c0b0884c4e17776f1d02353a8d83e89fed06d3)
@@ -1,6 +0,0 @@
-package it.polimi.ingsw.galaxytruckers.view.enums;
-
-public enum RotationDirection {
-    LEFT,  // Represents a 90° counter-clockwise rotation
-    RIGHT  // Represents a 90° clockwise rotation
-}
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ExternalStorageConfigurationManager\" enabled=\"true\" />\r\n  <component name=\"MavenProjectsManager\">\r\n    <option name=\"originalFiles\">\r\n      <list>\r\n        <option value=\"$PROJECT_DIR$/pom.xml\" />\r\n      </list>\r\n    </option>\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_X\" default=\"true\" project-jdk-name=\"openjdk-24\" project-jdk-type=\"JavaSDK\" />\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 61c0b0884c4e17776f1d02353a8d83e89fed06d3)
+++ b/.idea/misc.xml	(date 1750668562440)
@@ -8,5 +8,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_X" default="true" project-jdk-name="openjdk-24" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_24" default="true" project-jdk-name="openjdk-24" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
 </project>
\ No newline at end of file
